src/kvilib/core/KviShortcut.h

```diff
@@ -48,6 +48,7 @@ class KVILIB_API KviShortcut
 #else
 	static QShortcut * create(const char * key, QObject * parent, const char * member = nullptr, const char * ambiguousMember = nullptr, Qt::ShortcutContext context = Qt::WindowShortcut);
 	static QShortcut * create(const QKeySequence & key, QObject * parent, const char * member = nullptr, const char * ambiguousMember = nullptr, Qt::ShortcutContext context = Qt::WindowShortcut);
+	static QShortcut * create(const QKeySequence::StandardKey & key, QObject * parent, const char * member = nullptr, const char * ambiguousMember = nullptr, Qt::ShortcutContext context = Qt::WindowShortcut);
```

Comment:
It's declared but undefined in case of Qt 5?

Reply:
It's defined and implemented only for Qt6. On a second view it seems correct to me, maybe I'm missing something?

Reply:
Right, I misread placement of ifdefs